tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -141,3 +147,20 @@ def test_return_url_per_environment(self):
         expected_url = "http://testserver:80{}".format(url)
         returned_url = return_url_per_environment(url)
         self.assertEqual(returned_url, expected_url)
+
+
+class CreateDirAsPythonPackageTest(BaseTestClass):
+    def setUp(self):
+        super(CreateDirAsPythonPackageTest, self).setUp()
+
+        self.BASE_TEMP_DIR = tempfile.mkdtemp()
+        self.temp_directory = join(self.BASE_TEMP_DIR, "temp_dir")
+
+    def test_create_dir_as_python_package(self):
+        create_dir_as_python_package(self.temp_directory)
+        self.assertTrue(os.path.isfile(join(self.temp_directory, "__init__.py")))
+
+        with open(join(self.temp_directory, "__init__.py")) as f:
+            self.assertEqual(f.read(), "")
+
+        shutil.rmtree(self.temp_directory)
```

Comment:
@takitsuse can you add a check to test whether folder is deleted or not.

Reply:
File deletion is not what `create_dir_as_python_package` does, so I would say we don't have to test it...

Reply:
Yeah I understood that, but next time when this test runs and remove dir fails second run can give a false positive as the folder already exists

Reply:
Now I see what you meant. will change that.